tests/gordo_components/client/test_client.py

```diff
@@ -63,8 +60,9 @@ def test_client_download_model(watchman_service):
 
 @pytest.mark.parametrize("batch_size", (10, 100))
 @pytest.mark.parametrize("use_data_provider", (False, True))
+@pytest.mark.parametrize("use_parquet", (True, False))
 def test_client_predictions_diff_batch_sizes_and_toggle_data_provider(
-    influxdb, watchman_service, use_data_provider: bool, batch_size: int
+    influxdb, watchman_service, use_data_provider: bool, batch_size: int, use_parquet
```

Comment:
Minor thing, but perhaps have ```use_parquet: bool``` instead

Reply:
Nice catch, I would have made the same comment. :steam_locomotive: